apps/src/util/formatValidation.js

```diff
@@ -1,14 +1,19 @@
 // RFC 5322 Official Standard email regex
 // Source: http://emailregex.com/
+
```

Comment:
This intentional? I think we don't want it, to be clear that the above comment pertains to this regex on the next line.